charts/datadog/templates/_container-otel-agent.yaml

```diff
@@ -0,0 +1,80 @@
+{{- define "container-otel-agent" -}}
+- name: otel-agent
+  image: "{{ include "image-path" (dict "root" .Values "image" .Values.agents.image) }}"
```

Comment:
do we plan to use the same image that for the other agent container?

Reply:
yes.  the agent image will include OTel -Agent.  See PR https://github.com/DataDog/datadog-agent/pull/26770. It will have a different tag